### PR TITLE
Minor mods to the landing

### PR DESCRIPTION
**Pull request by adityadipankar**

### CODE REVIEW

**Comment by adityadipankar** [comment]
1. Added some style attributes
2. Edited text on welcome screen

**Comment by planemad** [comment]
@adityadipankar the html links to `https://fonts.googleapis.com/css?family=Anton&display=swap`, but the css is referring to Roboto. Is that expected?

**Comment by adityadipankar** [comment]
The Anton could be removed @planemad 
I forgot to remove it. Will clean up in next push.

File: client/src/index.css [hunk 1]
```diff
@@ -13,13 +13,24 @@
   src: url(./fonts/BarlowSemiCondensed-Black.ttf) format('truetype');
   font-weight: 900
 }
-
+<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700,900|Signika:300,400,600,700&display=swap" rel="stylesheet">
```
Comment (planemad) [inline client/src/index.css]: Should this be in the html?

File: client/public/index.html [hunk 2]
```diff
@@ -11,6 +11,8 @@
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link href="https://fonts.googleapis.com/css?family=Anton&display=swap" rel="stylesheet">
```
Comment (planemad) [inline client/public/index.html]: Anton doesnt seem to be used in the CSS

File: client/src/index.css (hunk 1 above)
Comment (kshivanku) [reply client/src/index.css]: The best way to include files in a react project would be to host them on out server. This has to do with how webpack works. See this: https://stackoverflow.com/questions/41676054/how-to-add-fonts-to-create-react-app-based-projects

File: client/src/components/IntroScreen.js [hunk 3]
```diff
@@ -10,22 +10,23 @@ export function IntroScreen() {
...
-            <motion.div 
+            <motion.div
                 className="introScreenContent"
                 animate={contentControls}
             >
-                <h1>This is a visualization of all the protests against Citizenship Amendment Act (CAA) across India</h1>
-                <motion.button 
+                <h1>India has been on the streets this winter to save the secular fabric of the country</h1>
+                <h2>Over 20 civilians have died since these protests began on 20th of December 2019. Here is a map of the protests across the country.</h2>
+                <motion.button
```
Comment (kshivanku) [inline client/src/components/IntroScreen.js]: This much text is giving a scroll on mobile and pushing the CTA below scroll, can we reduce this?

Comment (kshivanku) [reply client/src/components/IntroScreen.js]: Actually scratch that

**Comment by kshivanku** [comment]
Made all the changes suggested in the pull request. Thank you Aditya, I'll close this without merging.

**Pull request closed without merging**